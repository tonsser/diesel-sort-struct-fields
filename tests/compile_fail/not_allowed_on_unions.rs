use diesel_sort_struct_fields::sort_fields;

#[sort_fields]
pub union Thing {
    a: u32,
}

fn main() {}
