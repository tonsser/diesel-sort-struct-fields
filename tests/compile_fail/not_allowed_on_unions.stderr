error: `#[sort_fields]` is not allowed on unions, only structs
 --> $DIR/not_allowed_on_unions.rs:4:5
  |
4 | pub union Thing {
  |     ^^^^^
